### PR TITLE
[orc8r][LI] ] Support X1 interface APIs in swagger

### PR DESCRIPTION
**Pull request by ymasmoudi**
Signed-off-by: YOUSSEF EL MASMOUDI <ymasmoudi@fb.com>

## Summary

This PR implements both X1 Tasks and Destinations to  manage interception requests

Screenshot 2021-02-11 at 18 32 34

## Test Plan

Unit test and local orc8r

### CODE REVIEW

**Comment by ymasmoudi** [comment]
cc @ulaskozat @koolzz

**Comment by ulaskozat** [comment]
@ymasmoudi can we make these API end points more generic? Let's conceptualize them as probes at the subscriber level, AGW level and network level.

File: lte/cloud/go/lte/const.go [hunk 1]
```diff
@@ -74,6 +74,8 @@ const (
 	PolicyRuleEntityType          = "policy"
 	RatingGroupEntityType         = "rating_group"
 	SubscriberEntityType          = "subscriber"
+	X1TaskEntityType              = "x1_task"
+	X1DestinationEntityType       = "x1_destination"
```
Comment (ulaskozat) [inline lte/cloud/go/lte/const.go:78]: Instead of X1, we can use NetworkProbe.

File: lte/cloud/go/services/lte/obsidian/handlers/handlers.go [hunk 2]
```diff
@@ -85,6 +85,14 @@ const (
...
 	ManageEnodebPath   = ListEnodebsPath + obsidian.UrlSep + ":enodeb_serial"
 	GetEnodebStatePath = ManageEnodebPath + obsidian.UrlSep + "state"
+
+	X1Tasks                        = "x1_tasks"
+	ManageNetworkX1TasksPath       = ManageNetworkPath + obsidian.UrlSep + X1Tasks
+	ManageNetworkX1TaskDetailsPath = ManageNetworkX1TasksPath + obsidian.UrlSep + ":task_id"
+
+	X1Destinations                        = "x1_destinations"
+	ManageNetworkX1DestinationsPath       = ManageNetworkPath + obsidian.UrlSep + X1Destinations
+	ManageNetworkX1DestinationDetailsPath = ManageNetworkX1DestinationsPath + obsidian.UrlSep + ":destination_id"
```
Comment (ulaskozat) [inline lte/cloud/go/services/lte/obsidian/handlers/handlers.go:95]: X1 -> NetworkProbe
x1_tasks -> network_probe_tasks
x1_destinations -> network_probe_destinations

File: lte/cloud/go/services/lte/obsidian/handlers/handlers.go [hunk 3]
```diff
@@ -126,6 +134,20 @@ func GetHandlers() []obsidian.Handler {
...
+		{Path: ManageNetworkX1TaskDetailsPath, Methods: obsidian.GET, HandlerFunc: getX1Task},
+		{Path: ManageNetworkX1TaskDetailsPath, Methods: obsidian.PUT, HandlerFunc: updateX1Task},
+		{Path: ManageNetworkX1TaskDetailsPath, Methods: obsidian.DELETE, HandlerFunc: deleteX1Task},
+
+		// LI X1 Destination
+		{Path: ManageNetworkX1DestinationsPath, Methods: obsidian.GET, HandlerFunc: listX1Destinations},
+		{Path: ManageNetworkX1DestinationsPath, Methods: obsidian.POST, HandlerFunc: createX1Destination},
+		{Path: ManageNetworkX1DestinationDetailsPath, Methods: obsidian.GET, HandlerFunc: getX1Destination},
+		{Path: ManageNetworkX1DestinationDetailsPath, Methods: obsidian.PUT, HandlerFunc: updateX1Destination},
+		{Path: ManageNetworkX1DestinationDetailsPath, Methods: obsidian.DELETE, HandlerFunc: deleteX1Destination},
```
Comment (ulaskozat) [inline lte/cloud/go/services/lte/obsidian/handlers/handlers.go:150]: X1 -> NetworkProbe
ManageNetworkX1 -> ManageNetworkProbe

**Pull request closed without merging**